packages/aws-cdk/test/api/hotswap/hotswap-deployments.test.ts

```diff
@@ -3,15 +3,25 @@ import * as setup from './hotswap-test-setup';
 
 let hotswapMockSdkProvider: setup.HotswapMockSdkProvider;
 let mockUpdateLambdaCode: (params: Lambda.Types.UpdateFunctionCodeRequest) => Lambda.Types.FunctionConfiguration;
+let mockTagResource: (params: Lambda.Types.TagResourceRequest) => {};
+let mockUntagResource: (params: Lambda.Types.UntagResourceRequest) => {};
+let mockGetFunctionConfiguration: (params: Lambda.Types.GetFunctionConfigurationRequest) => Lambda.Types.FunctionConfiguration;
 let mockUpdateMachineDefinition: (params: StepFunctions.Types.UpdateStateMachineInput) => StepFunctions.Types.UpdateStateMachineOutput;
 let mockGetEndpointSuffix: () => string;
 
 beforeEach(() => {
   hotswapMockSdkProvider = setup.setupHotswapTests();
   mockUpdateLambdaCode = jest.fn();
+  mockTagResource = jest.fn();
+  mockUntagResource = jest.fn();
+  mockGetFunctionConfiguration = jest.fn(({ FunctionName }) => {
+    return {
+      FunctionArn: `aws:::${FunctionName}`,
+    };
+  });
   mockUpdateMachineDefinition = jest.fn();
   mockGetEndpointSuffix = jest.fn(() => 'amazonaws.com');
-  hotswapMockSdkProvider.setUpdateFunctionCodeMock(mockUpdateLambdaCode);
+  hotswapMockSdkProvider.setUpdateFunctionCodeMock(mockUpdateLambdaCode, mockTagResource, mockUntagResource, mockGetFunctionConfiguration);
   hotswapMockSdkProvider.setUpdateStateMachineMock(mockUpdateMachineDefinition);
```

Comment:
Since w're not referencing these variables (`mockTagResource`, `mockUntagResource`, `mockUntagResource`, `mockGetFunctionConfiguration`) anywhere, can we please have `setUpdateFunctionCodeMock()` just set them for us to default values (just simple Jest mocks are fine)?

Reply:
The thing is, we _do_ reference them later in the tests. How would you rewrite the tests to do this? (see for example around lines 66-67)

Reply:
Not sure what you mean. This is the entire diff in this file:
<img width="1135" alt="Screen Shot 2021-12-08 at 14 33 56" src="https://user-images.githubusercontent.com/460937/145302125-f25b251f-139f-490f-ae7d-e16a62ca7a0b.png">

Reply:
Oh, I see! Yeah, can do.